File: apzkr-pzpi-21-11-milennyi-ivan/Task3-WebClient/src/viewModels/ShepherdVM.ts
// Клас для відображення інформації про пастуха
export class ShepherdVM {
    id: number; // Ідентифікатор пастуха
    name: string; // Ім'я пастуха
    surname: string; // Прізвище пастуха

    constructor(id: number, name: string, surname: string) {
        this.id = id;
        this.name = name;
        this.surname = surname;
    }
}

File: apzkr-pzpi-21-11-milennyi-ivan/Task1-Server/src/view_models/shepherd_vm.rs
use serde::Serialize;
use sqlx::FromRow;
use utoipa::ToSchema;

// Структура, що представляє представлення пастуха (ViewModel)
#[derive(Debug, Serialize, FromRow, ToSchema)]
pub(crate) struct ShepherdVM {
    id: u64, // Ідентифікатор пастуха
    name: String, // Ім'я пастуха
    surname: String, // Прізвище пастуха
}
